radis/lbl/loader.py

```diff
@@ -931,6 +931,7 @@ def fetch_databank(
         load_columns="equilibrium",
         parallel=True,
         extra_params=None,
+        verbose=True,
```

Comment:
There is a SpectrumFactory.verbose parameter, no need to create a new parameter in the function

Just use 
```
if self.verbose
```
or 
```
verbose = self.verbose
if verbose ... 
```

(also, not related but always good to mention, we can have multiple verbose levels like 
```
if verbose >= 2: print("some extra details")
```

Reply:
self.verbose ==> indeed, done